src/features/lightspeed/playbookExplanation.ts

```diff
@@ -43,21 +44,27 @@ export const playbookExplanation = async (
   const lightSpeedStatusbarText =
     await lightSpeedManager.statusBarProvider.getLightSpeedStatusBarText();
 
-  const accessToken =
-    await lightspeedAuthenticatedUser.getLightspeedUserAccessToken();
   let markdown = "";
   lightSpeedManager.statusBarProvider.statusBar.text = `$(loading~spin) ${lightSpeedStatusbarText}`;
   try {
-    const response: ExplanationResponse = await client.sendRequest(
-      "playbook/explanation",
-      {
-        accessToken: accessToken,
-        URL: settingsManager.settings.lightSpeedService.URL,
-        content: content,
-        explanationId: explanationId,
-      },
-    );
-    markdown = response.content;
+    generateExplanation(
+      content,
+      explanationId,
+      client,
+      lightspeedAuthenticatedUser,
+      settingsManager,
+    ).then((response: ExplanationResponse) => {
+      if (isError(response)) {
+        vscode.window.showErrorMessage(response.message ?? UNKNOWN_ERROR);
+        currentPanel.setContent(
+          `<p><span class="codicon codicon-error"></span>Cannot load the explanation: <code>${response.code}</code></p>`,
```

Comment:
Since is a message that is visible by the user. Something like: `The operation has failed:` would be better.